src/main/java/com/commercetools/sync/CliRunner.java

```diff
@@ -0,0 +1,152 @@
+            .longOpt(SYNC_MODULE_OPTION_LONG)
+            .desc(SYNC_MODULE_OPTION_DESCRIPTION)
+            .hasArg()
+            .build();
+
+    final Option helpOption =
+        Option.builder(HELP_OPTION_SHORT)
+            .longOpt(HELP_OPTION_LONG)
+            .desc(HELP_OPTION_DESCRIPTION)
+            .build();
+
+    final Option versionOption =
+        Option.builder(VERSION_OPTION_SHORT)
+            .longOpt(VERSION_OPTION_LONG)
+            .desc(VERSION_OPTION_DESCRIPTION)
+            .build();
+    options.addOption(syncOption);
+    options.addOption(helpOption);
+    options.addOption(versionOption);
+    return options;
+  }
+
+  void run(@Nonnull final String[] arguments) {
+    final CommandLineParser parser = new DefaultParser();
+    try {
+      final CommandLine commandLine = parser.parse(getOptions(), arguments);
+      processCliArguments(commandLine);
+    } catch (final ParseException | IllegalArgumentException exception) {
+      handleIllegalArgumentException(format("Parse error:%n%s", exception.getMessage()));
+    }
+  }
+
+  private void processCliArguments(@Nonnull final CommandLine commandLine) {
+    final Option[] options = commandLine.getOptions();
+    if (options.length == 0) {
+      handleIllegalArgumentException("Please pass at least 1 option to the CLI.");
+    } else {
+      final Option option = options[0];
+      final String optionName = option.getOpt();
+      switch (optionName) {
+        case SYNC_MODULE_OPTION_SHORT:
+          processSyncOption(commandLine).toCompletableFuture().join();
+          break;
+        case HELP_OPTION_SHORT:
+          printHelpToStdOut();
+          break;
+        case VERSION_OPTION_SHORT:
+          logApplicationVersion();
+          break;
+        default:
+          // Unreachable code since this case is already handled by parser.parse(options,
+          // arguments);
+          // in the constructor.
+          throw new IllegalStateException(format("Unrecognized option: -%s", optionName));
+      }
+    }
+  }
+
+  private void handleIllegalArgumentException(@Nonnull final String errorMessage) {
+    LOGGER.error(errorMessage);
+    printHelpToStdOut();
+  }
+
+  CompletionStage processSyncOption(@Nonnull final CommandLine commandLine) {
+    final String syncOptionValue = commandLine.getOptionValue(SYNC_MODULE_OPTION_SHORT);
+    return SyncerFactory.getSyncer(syncOptionValue).sync();
+  }
+
+  private void printHelpToStdOut() {
+    final HelpFormatter formatter = new HelpFormatter();
+    formatter.printHelp(getApplicationName(), getOptions());
+  }
+
+  private static String getApplicationName() {
+    final String implementationTitle = Main.class.getPackage().getImplementationTitle();
+    return isBlank(implementationTitle) ? APPLICATION_DEFAULT_NAME : implementationTitle;
+  }
+
+  private static void logApplicationVersion() {
+    final String implementationVersion = getApplicationVersion();
+    LOGGER.info(implementationVersion);
```

Comment:
Looks a bit strange 😫 
`[main] INFO com.commercetools.sync.CliRunner - 1.0.0`

Reply:
What do you mean? For this log message only? or for all log messages?

Reply:
Not for all, I think the version should be meaningful to the user. Now, It looks like a log message.

for example, the message of this call should be a simple stdout message, wdyt?

```bash
-> java -jar commercetools-sync.jar -v

-> 1.0.0
```

Reply:
I changed log messages to printing to console where needed: dfad69a